Week5/Week5.py

```diff
@@ -0,0 +1,31 @@
+# Write a python program create csv file student.csv(sid,sname,city,contact) using 10 records 
+##(in which 5 records input directly and 5 records take input from user). write records into this file.
+
+import csv
+p=open('E:\\22BCA245\\student3.csv','w',newline='')

```

Comment:
explain the second method to opening file...

Reply:
Second method to open the file is **withopen** . For example:-with open('E:\\22BCA245\\student3.csv','w',newline='') as f**(variable name.)**